README.md

```diff
@@ -50,26 +50,44 @@ When data scientists have refined their models to a point where they are ready t
 
 ## Getting Started
 
-To get started with **Arangoipe** using a [Jupyter Notebook](https://jupyter.org/), do the following:
+To facilitate an easy start, docker containers for *torch* and *tensorflow* are provided. These containers have all components of the **ArangoML Pipeline** (Arangopipe, ArangoDB and the Arangopipe user interface). To get started:
 
 1. Install [_docker_](https://docs.docker.com/install/)
 
 
-2. Start **Arangopipe** together with [Jupyter](https://jupyter.org/) by running:
+2. Start the container of your choice (*tensorflow* or *torch*):
 
     ` docker run -p 6529:8529 -p 8888:8888 -p 3000:3000 -it arangopipe/ap_tensor_flow`  (_tensorflow_)
     
     or, for _torch_,
     
     ` docker run -p 6529:8529 -p 8888:8888 -p 3000:3000 -it arangopipe/ap_torch`
 
+
+
+3. Running an example in the *torch* container: Run the *torch* docker container. You will have to use the `docker ps` command and get to the shell in the container using the `docker exec` command. Change directory to the `examples/pytorch` directory. The *torch* container provides an example of a linear regression model that uses **Arangopipe** to log experiment metadata. The experiment meta data includes information about the dataset, featureset and optimization settings used to run the *pytorch* model. Once you are in the shell of the *torch* container, run the driver program that develops the torch model and logs the experiment meta-data to *arangopipe*. The details are shown in the figure below.
```

Comment:
Can you provide the actual commands here?

Reply:
Sure the actual commands are in the screen shot below, but I can add the commands there.